config/config.go

```diff
@@ -52,6 +52,9 @@ type Config struct {
 	// AppHash changes. Defaults to true.
 	CreateEmptyBlocks bool
 
+	// MempoolWalDir sets directory path of mempool.wal. If empty, mempool.wal is disabled
+	MempoolWalDir string
```

Comment:
I think we should just have `MempoolWalEnabled bool`, and it should be `false` by default

Reply:
Done